github/retag-and-push.sh

```diff
@@ -31,8 +31,12 @@ fi
 if [[ "$GITHUB_REF" == "refs/heads"* ]]; then
     # push `branch-sha` tagged image
     branch="${GITHUB_REF/refs\/heads\//}"
-    docker tag "$input_image" "${name}:${branch}-${GITHUB_SHA}"
-    docker push "${name}:${branch}-${GITHUB_SHA}"
+    timestamp=$(date +%Y%m%d.%H%M)
```

Comment:
Can we have a `-u` to have UTC

Reply:
Good point, be explicit over relying on the local configuration. Servers should always be on UTC.

Reply:
Good catch. Done.